src/tools/chdman.cpp

```diff
@@ -2392,7 +2500,7 @@ static void do_extract_cd(parameters_map &params)
 	cdrom_file *cdrom = cdrom_open(&input_chd);
 	if (cdrom == nullptr)
 		report_error(1, "Unable to recognize CHD file as a CD");
-	const cdrom_toc *toc = cdrom_get_toc(cdrom);
+	cdrom_toc toc = *(cdrom_get_toc(cdrom));
```

Comment:
Why the extraneous parentheses?

Reply:
Avoids ambiguity with (*func)(). Compilers know the right order. Humans aren’t compilers.